package.json

```diff
@@ -9,13 +9,12 @@
     "font-awesome": "^4.7.0",
     "gulp": "^4.0.2",
     "gulp-if": "^3.0.0",
-    "gulp-less": "^4.0.1",
     "gulp-rename": "^2.0.0",
-    "gulp-sass": "^4.1.0",
+    "gulp-sass": "^5.0.0",
     "gulp-sourcemaps": "^2.6.5",
     "jquery": "^3.6.0",
-    "less": "^3.13.1",
     "moment": "^2.29.1",
+    "node-sass": "^6.0.1",
```

Comment:
It would be better to use sass instead of node-sass, libraries used by node-sass are deprecated already:

![image](https://user-images.githubusercontent.com/830663/125757666-05c1fa74-c3c0-472c-bfda-2562d2526f0b.png)

Reply:
@Zharktas There was some warning message thrown for `sass` so, I've added `node-sass. I think bootstrap 3.4 is compatible only with `node-sass.  We can change it later while upgrading the bootstrap 5.

Reply:
They are only deprecation warnings, it still compiles division as before. The warnings should go away when upgraded to bootstrap 5. I would be okay with the warnings compared to using deprecated versions.

Reply:
Sounds good. Replaced with  `sass`.

Reply:
Hi @Zharktas,   is there anything else to improve on this ?